cli/cmd/plugin/diagnostics/e2e-test/e2e_test.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
```

Comment:
`ioutil` should not be used. Will have to check why the linter didn't complain about this.

Reply:
Yea, that's deprecated... I wonder why it didn't say anything 🤔

Reply:
Thanks @stmcginnis and @dvonthenen for reviewing. I will replace it.